Gems/Atom/Feature/Common/Code/Include/Atom/Feature/Debug/RenderDebugFeatureProcessorInterface.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <Atom/RPI.Public/FeatureProcessor.h>
+#include <Atom/Feature/Debug/RenderDebugSettingsInterface.h>
+
+namespace AZ {
+    namespace Render {
+
+        //! TODO
```

Comment:
Minor - might want to add this comment now.

Reply:
Good catch, thanks!